### DISCUSSION TITLE
[Detect:Warning]	Another tablet driver found: Wacom (but i dont)

### DISCUSSION THREAD

**Discussion by MyAzr** [OpenTabletDriver/OpenTabletDriver#2467] [Q&A] [answered]
I deleted wacom driver. And yes I restarted, installed, deleted again but nothing is working 
Have I done something wrong?

(in folder System32 only undeletable "wacompen.sys" file)
![image](https://user-images.githubusercontent.com/67195203/197444957-5040075a-9c66-440e-9f4a-7a8a1535bcc8.png)
![image](https://user-images.githubusercontent.com/67195203/197445418-77f9aaa1-4556-4458-8e08-414b8ba0eb6f.png)

**Comment by X9VoiD** [comment, marked as answer]
Do **not** delete. Properly uninstall the driver. Just deleting the driver leaves behind things that interfere with OTD.

**Reply by MyAzr** [reply]
by delete I mean uninstall. (English is not my 1st or 2nd  language)

**Reply by X9VoiD** [reply]
Snippet from OTD discord server:

In the drop down container, find any of the following and right click them
- <OEM> tablet (e.g. “Wacom Tablet”; “Gaomon Tablet”)
- the model number (e.g. “CTL-472”)
- the name of the tablet (“Wacom Intuos S”)
Then click on update -> browse my computer for drivers -> let me pick from a list of available drivers -> usb input/composite device -> next

**Reply by MyAzr** [reply]
that works, but it's still says that warning.
but it works. THX! i guess im just gonna ignore that warning(?)
![image](https://user-images.githubusercontent.com/67195203/197473050-54ad0941-d27e-4352-a72c-543da790c1c9.png)
